target_chains/ethereum/entropy_sdk/solidity/EntropyStructs.sol

```diff
@@ -59,8 +59,8 @@ contract EntropyStructs {
         address requester;
         // If true, incorporate the blockhash of blockNumber into the generated random value.
         bool useBlockhash;
-        // If true, the requester will be called back with the generated random value.
-        bool isRequestWithCallback;
-        // There are 2 remaining bytes of free space in this slot.
+        // Status flag for requests with callbacks. See EntropyConstants for the possible values of this flag.
+        uint8 status;
```

Comment:
(bools are represented as uint8 so this is fine. I'm reusing this flag because I want the remaining 2 bytes for the gas limit)